modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -48,6 +48,73 @@
 #include <shlwapi.h>
 #endif
 
+static bool _get_blender_version(const String &p_path, int &major, int &minor, String *r_err = nullptr) {
+	String path = p_path;
+#ifdef WINDOWS_ENABLED
+	path = path.path_join("blender.exe");
```

Comment:
I believe on Windows `.exe` is optional.

Reply:
I think so, but this is how we get unrelated fixes with git blame associated with your name. I think it's better that multiple reviews find new things and result in fixes applied.

Reply:
I removed the upwards gate, but I didn't make the Windows fix, since I didn't have a good way to test it.